app/src/main/java/org/openimis/imispolicies/Statistics.java

```diff
@@ -322,33 +334,46 @@ private void GetEnrolmentStats(){
         } catch (ParseException e) {
             e.printStackTrace();
         }
-        String stats = null;
 
-        cs.setFunctionName("GetEnrolmentStats");
-        stats = cs.GetEnrolmentStats(OfficerCode, FromDate, ToDate);
+        DateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
I know that openIMIS is using 'dd/MM/yyyy' date format. Are you sure this is the right format?